src/label/creator.py

```diff
@@ -203,3 +202,79 @@ def create_box_label(member_number, name):
                 font=name_font)
 
     return label
+
+
+def create_chemical_storage_label(member_id, name):
+
+    storage_text = 'This product belongs to'
+    storage_text_size, storage_font = get_font_size(75, storage_text)
+
+    id_text = f'#{member_id}'
+    id_text_size, id_font = get_font_size(300, id_text)
+
+    name_text = name
+    name_text_size, name_font = get_font_size(200, name_text)
+
+    instruction_text = f'Any member can use this product after'
+    instruction_text_size, instruction_font = get_font_size(50, instruction_text)
+
+    date_text = get_end_date_string(CHEMICAL_STORAGE_LENGTH)
+    date_text_size, date_font = get_font_size(300, date_text)
+
+    label = Image.new('RGB',
```

Comment:
Maybe rename the variable in this function as well, e.g.
```suggestion
    label_image = Image.new('RGB',
```